.github/workflows/docs.yaml

```diff
@@ -22,10 +22,10 @@ jobs:
                 with:
                     token: ${{ secrets.APIFY_SERVICE_ACCOUNT_GITHUB_TOKEN }}
 
-            -   name: Use Node.js 16
-                uses: actions/setup-node@v2-beta
```

Comment:
Why a beta? :smile:

Reply:
no idea, this has been untouched for years apparently, it should be updated by renovate too